invenio_records_rest/facets.py

```diff
@@ -38,6 +38,27 @@ def inner(values):
     return inner
 
 
+def range_filter(field, **kwargs):
+    """Create a range filter."""
+    def inner(values):
+        range_ends = [int(val) for val in values[0].split(',')]
```

Comment:
- [ ] We should be able to support not only `int`.
- [ ] Why using `,` as separator for ranges?
- [ ] Have you considered using Elasticsearch-DSL `Range` instead?